src/briefcase/platforms/web/static.py

```diff
@@ -0,0 +1,347 @@
+                    )
+                    css_file.write(f" * {package}::{'/'.join(path.parts[2:])}\n")
+                    css_file.write(
+                        " *******************************************************/\n\n"
+                    )
+                    css_file.write(wheel.read(filename).decode("utf-8"))
+
+    def build_app(self, app: AppConfig, **kwargs):
+        """Build the static web deployment for the application.
+
+        :param app: The application to build
+        """
+        self.logger.info("Building web project...", prefix=app.app_name)
+
+        if self.wheel_path(app).exists():
+            with self.input.wait_bar("Removing old wheels..."):
+                self.tools.shutil.rmtree(self.wheel_path(app))
+
+        self.wheel_path(app).mkdir(parents=True)
+
+        with self.input.wait_bar("Building app wheel..."):
+            try:
+                self.tools.subprocess.run(
+                    [
+                        sys.executable,
+                        "-u",
+                        "-m",
+                        "wheel",
+                        "pack",
+                        self.app_path(app),
+                        "--dest-dir",
+                        self.wheel_path(app),
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to build wheel for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Installing wheels for dependencies..."):
+            try:
+                self.tools.subprocess.run(
+                    [
+                        sys.executable,
+                        "-u",
+                        "-m",
+                        "pip",
+                        "wheel",
+                        "--wheel-dir",
+                        self.wheel_path(app),
+                        "-r",
+                        self.bundle_path(app) / "requirements.txt",
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to install dependencies for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Writing Pyscript configuration file..."):
+            with (self.project_path(app) / "pyscript.toml").open(
+                "w", encoding="utf-8"
+            ) as f:
+                f.write(f"name = '{app.formal_name}'\n")
+                f.write(f"description = '{app.description}'\n")
+                f.write(f"version = '{app.version}'\n")
+                f.write("\n")
+                f.write("autoclose_loader = true\n")
+                f.write("packages = [\n")
+                f.write(
+                    "\n".join(
+                        f'    "/{wheel.relative_to(self.project_path(app))}",'
```

Comment:
Additionally, looks like pyscript et al are expecting the Unix path separator.

From browser console:
```
[pyscript/pyodide] micropip install /static\wheels\mywebapp-0.0.1-py3-none-any.whl,/static/wheels/toga_core-0.3.0.dev36-py3-none-any.whl,/static/wheels/toga_dummy-0.3.0.dev36-py3-none-any.whl,/static/wheels/toga_web-0.3.0.dev36-py3-none-any.whl,/static/wheels/travertino-0.1.3-py3-none-any.whl [logger.ts:39:19](https://pyscript.net/src/logger.ts)
Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.10/asyncio/futures.py", line 201, in result
    raise self._exception
  File "/lib/python3.10/asyncio/tasks.py", line 234, in __step
    result = coro.throw(exc)
  File "/lib/python3.10/site-packages/micropip/_micropip.py", line 548, in install
    await transaction.gather_requirements(requirements)
  File "/lib/python3.10/site-packages/micropip/_micropip.py", line 305, in gather_requirements
    await gather(*requirement_promises)
  File "/lib/python3.10/asyncio/futures.py", line 284, in __await__
    yield self  # This tells Task to wait for completion.
  File "/lib/python3.10/asyncio/tasks.py", line 304, in __wakeup
    future.result()
  File "/lib/python3.10/asyncio/futures.py", line 201, in result
    raise self._exception
  File "/lib/python3.10/asyncio/tasks.py", line 232, in __step
    result = coro.send(None)
  File "/lib/python3.10/site-packages/micropip/_micropip.py", line 315, in add_requirement
    wheel = WheelInfo.from_url(req)
  File "/lib/python3.10/site-packages/micropip/_micropip.py", line 75, in from_url
    name, version, build, tags = parse_wheel_filename(file_name)
  File "/lib/python3.10/site-packages/packaging/utils.py", line 100, in parse_wheel_filename
    raise InvalidWheelFilename(f"Invalid project name: {filename}")
packaging.utils.InvalidWheelFilename: Invalid project name: static\wheels\mywebapp-0.0.1-py3-none-any
    PythonError https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    new_error https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    method_call_trampoline https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    callPyObjectKwargs https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    callPyObject https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    wrapper https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    setTimeout handler*hiwire_call_bound https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    callPyObjectKwargs https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    callPyObject https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    wrapper https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    setTimeout handler*hiwire_call_bound https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.asm.js:10
    ```

Reply:
ack - fixed.